Tests/SwiftKueryTests/TestUpdate.swift

```diff
@@ -79,4 +80,84 @@ class TestUpdate: XCTestCase {
         query = "DELETE FROM tableUpdate"
         XCTAssertEqual(kuery, query, "\nError in query construction: \n\(kuery) \ninstead of \n\(query)")
     }
+    
+    func testUpdateAndDeleteWith() {
+        let t = MyTable()
+        let t2 = MyTable2()
+        let connection = createConnection()
+        let psqlConnection = createConnection(withDeleteRequiresUsing: true, withUpdateRequiresFrom: true)
```

Comment:
Could you please rename this, maybe just `connection2`. I prefer to avoid referring specific dbs